### DISCUSSION TITLE
Capture and Transmit work fine, but signal not decoded

### DISCUSSION THREAD

**Discussion by lpmlpm** [bengtmartensson/IrScrutinizer#446] [Decoding issues] [answered]
I have a IR Toy 2 connected to a Windows 10 laptop . IR toy 2 passes self-diagnostics.

I am able to capture signals in IRScruitinizer and transmit the capture signal. There are no errors. But the "Decode" field remains empty. I don't understand. I know the signal is successfully captured since it is able to operate my TV when re-transmitted from IRScruitinizer.

I attach a screenshot and the captured signal that won't decode.

[IrScrutinizer.zip](https://github.com/bengtmartensson/IrScrutinizer/files/8043786/IrScrutinizer.zip)

**Comment by bengtmartensson** [comment, marked as answer]
It appears that the frequency delivered from the IrToy is off (is 41870, should probably be around 38000). The IrToy, despite of having a QSE157, often produces bad frequency measurements. You can try to reduce the distance between QSE and remote down to 1cm (1/2"); it may help.

To decode the actually captured signal, you can set a higher frequency tolerance in IrScrutinizer (Options -> Protocol Parameters -> Frequency Tolerance (Hz)). Set it to either 4000 or -1 (meaning do not check frequency at all). 

The signal will then decode.

**Reply by lpmlpm** [reply]
Thanks, you nailed the issue.

I found out that if I lower the captured frequency from 42000 Hz to 38000 Hz then it decodes fine.
I increased frequency tolerance to 5000 Hz and now it is working fine.

Reducing distance to 1cm didn't help.
